src/diode.js

```diff
@@ -4,9 +4,16 @@
  * that state has changed.
  */
 
+/**
+ * Important: 120 FPS is granular enough to get around differences
+ * in the animation and time clock
+ */
+var FRAMES = 1000 / 120
+
 function Diode(target) {
   var _callbacks = []
   var _tick      = target
+  var _lastFire  = null
```

Comment:
If we moved this outside of the closure, we could allow all `volley` events to synchronize to the same clock. Is this desireable?